agents/components/MemberInvites.js

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { connect as connectFela } from 'react-fela'
+import { Field, FieldArray, reduxForm as connectForm } from 'redux-form'
+import { flow } from 'lodash'
+import { TextField } from 'redux-form-material-ui'
+
+import styles from '../styles/MemberInvites'
+import Button from '../../app/components/Button'
+
+function renderMembers ({ fields, meta: { error, submitFailed } }) {
+  return (
+    <div>
+      <Button type='button' onClick={() => fields.push({})}>Add Member</Button>
```

Comment:
so if we want to do the "next member field always showing" approach, then rather than add a button to create an empty member, we'll want to add an extra set of fields at the bottom of this render (as if it were an empty member), where somehow we're able to hook into when you start typing it will `push` the next member.